tools/cldr-code/src/main/java/org/unicode/cldr/icu/LDMLComparator.java

```diff
@@ -115,9 +115,11 @@ public static void main(String[] args) {
 
     static Collator getDefaultCollation() {
         // if (DEFAULT_COLLATION != null) return DEFAULT_COLLATION;
-        RuleBasedCollator temp = (RuleBasedCollator) Collator.getInstance(ULocale.ENGLISH);
+        RuleBasedCollator temp =
+                (RuleBasedCollator) Collator.getInstance(ULocale.ROOT); // freeze below
         temp.setStrength(Collator.IDENTICAL);
         temp.setNumericCollation(true);
+        temp = (RuleBasedCollator) temp.freeze();
```

Comment:
This collator could also be added to CollatorHelper, combining numeric and strength IDENTICAL